src/assets/sendGasLimits.ts

```diff
@@ -1,11 +1,30 @@
+import { ChainId } from '../types'
+import { supportsNonNativeAssets } from '../chains'
+
 const sendGasLimits = {
   BTC: 290,
   NATIVE_EVM: 21000, // EVM -> ETH, RBTC, MATIC, BNB, AVAX, FUSE
   ERC20_EVM: 90000, // EVM -> ETH, RBTC, MATIC, BNB, AVAX, FUSE
   TERRA: 100000, // applies on both native and ERC2 Terra assets
-  ARBETH: 620000,
+  ARBETH: 620000, // for native asset is around ~420k and for ERC20 ~540k
   NEAR: 10000000000000,
   SOL: 1000000
 }
 
-export { sendGasLimits }
+const getSendGasLimitERC20 = (chainId: ChainId): number | null => {
+  if (!supportsNonNativeAssets(chainId)) {
```

Comment:
I think it should rather throw error. You shouldn't be calling this method for a chain that doesn't support tokens